granulate_utils/linux/cgroups/cpu_cgroup.py

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+
+from granulate_utils.linux.cgroups.base_cgroup import BaseCgroup
+
+
+class CpuCgroup(BaseCgroup):
+    HIERARCHY = "cpu"
+
+    def set_cpu_limit_cores(self, cores: float) -> None:
+        period = int(self.read_from_controller("cpu.cfs_period_us").split("\n")[0])
+        self.write_to_controller("cpu.cfs_quota_us", str(period * cores))
+
+    def get_cpu_limit_cores(self) -> float:
+        period = float(self.read_from_controller("cpu.cfs_period_us").split("\n")[0])
```

Comment:
Why `float`? It's an integer

Reply:
what if we want 1.5 cores?

Reply:
The `period` is always an integer. `/` operator is true divide even if both operands are integers

Reply:
right, i though you referred to something else